src/models/StyledNativeComponent.js

```diff
@@ -163,6 +163,7 @@ export default (constructWithOptions: Function) => {
     } = options
 
     const inlineStyle = new InlineStyle(
+      StyleSheet,
```

Comment:
Instead of doing this, the `InlineStyle` model should become a factory function that accepts an RN/RP StyleSheet.

have look on how some other models do it :) This is the closest example obviously: https://github.com/mathieudutour/styled-components/blob/c65a1cb54d81eddbac48fa845627e48fb007d3ef/src/models/ComponentStyle.js#L11

So instead of passing your StyleSheet in, we can pass the generated InlineStyle in